CHANGELOG.md

```diff
@@ -46,32 +46,82 @@ DecryptBinaryMessageArmored(privateKey string, passphrase []byte, ciphertext str
 (key *Key) ToPublic() (publicKey *Key, err error) 
 ```
 
-- Helpers to handle detached signatures
+- Helpers to handle encryption (both with armored and unarmored cipher) + encrypted detached signatures in one call.
 ```go
 EncryptSignArmoredDetached(
-    publicKey, privateKey string,
-    passphrase, plainData []byte,
-) (ciphertext, signature string, err error)
+	publicKey, privateKey string,
+	passphrase, plainData []byte,
+) (ciphertextArmored, encryptedSignatureArmored string, err error)
```

Comment:
Wait, I've seen this today, but this is a breaking change.
@marinthiercelin @twiss we shouldn't be doing this